girdereegviewer/app/eeg/components.py

```diff
@@ -0,0 +1,78 @@
+import eegviz.lib.libpyeegviz as eegviz
+import numpy as np
+import os
+from PIL import Image
+from trame_rca.utils import AbstractWindow
+
+
+class EEGWindow(AbstractWindow):
+    def __init__(self, data_path, **kwargs):
+        if not os.path.exists(data_path):
+            raise Exception("Path does not exist")
+
+        self._context = eegviz.create(data_path)
+        self._events = ["MouseMove", "LeftButtonPress", "RightButtonPress", "KeyDown"]
```

Comment:
It's odd to not have any mouse button up... Maybe add a comment if it is expected